TriangleApp/app/src/main/java/triangle/triangleapp/helpers/CameraHelper.java

```diff
@@ -0,0 +1,29 @@
+package triangle.triangleapp.helpers;
+
+import android.content.Context;
+import android.content.pm.PackageManager;
+import android.hardware.Camera;
+import android.util.Log;
+
+/**
+ * Created by Kevin Ly on 6/12/2017.
+ */
+
+public class CameraHelper {
+    private static final String TAG = "CameraHelper";
+
+    public static boolean checkCameraHardware(Context context) {
+        return context.getPackageManager().hasSystemFeature(PackageManager.FEATURE_CAMERA);
+    }
+
+    public static Camera getCameraInstance(){
```

Comment:
Added comment to method: getCameraInstance.
 /**
     * Open the camera and get the camera instance.
     * @return Opened camera instance
     */